src/REGothEngine.hpp

```diff
@@ -7,6 +7,8 @@
 
 namespace REGoth
 {
+  class OriginalGameFiles;
```

Comment:
The forward declaration is not necessary, is it?

Reply:
Ah, I put it there before I removed the method in which you list all vdfs packages. Good catch, I'll remove it!